src/components/UI/Buttons/PrimaryButton/index.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+
+import { FC } from "react";
+import cn from "classnames";
+import { ButtonProps } from "@/interfaces/ButtonInterface";
+
+const PrimaryButton: FC<ButtonProps> = ({
+  children,
+  styles = "",
+  fullWidth = false,
```

Comment:
```suggestion
  isFullWidth = false,
```

usally boolean variables are starting with is/has naming.